### PR TITLE
Information Updated With XML2Csv and ExecuteForEachEntryOf

### PR DESCRIPTION
**Pull request by MarcDouguedroit**

### CODE REVIEW

File: completions-compute.txt [hunk 1]
```diff
@@ -12,6 +12,7 @@ ConversionType
 DivideBy
 EntryKey
 ExcludeRegExp
+Json2Csv
```
Comment (bertysentry) [inline completions-compute.txt]: Please remove. This is only for keywords **after** `Compute(n).`, like *Column*, *TranslationTable*, or in the case of *XML2CSV*: *EntryKey*, etc.

File: completions-compute.txt [hunk 2]
```diff
@@ -31,3 +32,4 @@ SubSeparators
 Substract
 TranslationTable
 ValueList
+XML2Csv
```
Comment (bertysentry) [inline completions-compute.txt]: Please remove

File: completions-compute-type.txt [hunk 3]
```diff
@@ -19,3 +19,4 @@ RightConcat
 Substract
 SubString
 Translate
+XML2Csv
```
Comment (bertysentry) [inline completions-compute-type.txt]: Should be either *Xml2Csv* or *XML2CSV* to be consistent.

As it's *Json2Csv*, I guess it's best to have *Xml2Csv*

File: grammars/hdf.cson [hunk 4]
```diff
@@ -502,6 +502,7 @@
                   |Substract
                   |SubString
                   |Translate
+                  |XML2Csv
```
Comment (bertysentry) [inline grammars/hdf.cson]: Xml2Csv

**Review by bertysentry — CHANGES_REQUESTED** [review]

File: completions-compute-type.txt (hunk 3 above)
Comment (MarcDouguedroit) [reply completions-compute-type.txt]: Done

**Review by bertysentry — APPROVED** [review]

**Pull request merged**